README.md

```diff
@@ -1 +1,43 @@
-# hw01
\ No newline at end of file
+# hw01
+
+### Introduction
+
+My name is Vanessa Guigon and I am from *Chicago, Illinois*. I
+am a rising third year at UChicago majoring in **Public Policy** 
+and **Political Science**. 
+
+### About Me
+
+#### Fun Facts:
+- I was born in Florida and I moved to Chicago when I was young.
+- I am an identical twin.
+  - I am older than her by *16 minutes*.
+- I am a Dual Citizen of the **US** and **France**.
+
+#### Photo:
+Below is an image of myself.
+
+![](Me.jpg)
+
+
+#### My Favorite Quotes:
+Michelle Obama in her book *Becoming*:
+
+>Your story is what you have, what you will always have. It is 
+something to own.
+
+Barack Obama:
+
+>We are the change that we seek.
```

Comment:
Nice use of quotes. Have you seen the Netflix movie? The sem coop appears in it.